resources/todomvc/vanilla-examples/javascript-es5/src/helpers.js

```diff
@@ -0,0 +1,52 @@
+/* global NodeList */
```

Comment:
I wonder why this wasn't detected as a file move... I assume this is the same content as before but let me know if it isn't

Reply:
🤷